kibana/README.md

```diff
@@ -55,7 +55,8 @@ helm install --name kibana elastic/kibana --version 7.2.0 --set imageTag=7.2.0
 | `kibanaConfig`            | Allows you to add any config files in `/usr/share/kibana/config/` such as `kibana.yml`. See [values.yaml](./values.yaml) for an example of the formatting.                                                                                                                                                                                                     | `{}`                                                                                                                      |
 | `podSecurityContext`      | Allows you to set the [securityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod) for the pod                                                                                                                                                                                             | `fsGroup: 1000`                                                                                                                      |
 | `securityContext`      | Allows you to set the [securityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container) for the container                                                                                                                                                                                             | `capabilities.drop:[ALL]`<br>`runAsNonRoot: true`<br>`runAsUser: 1000`                                                                                                                      |
-| `serviceAccount`          | Allows you to overwrite the "default" [serviceAccount](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) for the pod                                                                                                                                                                                                        | `[]`                                                                                                                      |
+| `serviceAccount.create`    | Create service account               | `true`                                                                                                                                   |
+| `serviceAccount.name`    | The name of the ServiceAccount to use. If not set and create is true, a name is generated using the fullname template               | ``                                                                                                                                   |
```

Comment:
`serviceAccount` is already the name. Moving it to `serviceAccount.name` is going to be a breaking change. To do this in a backwards compatible way I propose using the same setup that filebeat and metricbeat are using with `managedServiceAccount` determining whether or not to manage the actual service account.

Reply:
Alright I've made some adjustments, but FWIW I think it's worth making this breaking change, because now we have `serviceAccount.name` and `managedServiceAccount.name`(with `managedServiceAccount.create` set to `false`) that achieve the exact same thing.
I would really prefer going with original PR and sticking with what's standard in so many other charts.

Reply:
Updating values to match the current standard being used is not a good enough reason to make a breaking change. We can totally do it for the next major release but that is still a while off. Or worse, it will cause a subtle breakage for anyone upgrading and not reading the readme for every single release. I find it very important that users can continue to upgrade to the latest version of the chart to take advantage of new features and fixes without needing to review every single change.

Reply:
Fair enough!